### ISSUE TITLE
Skip `open` if program not found

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1142]
At the moment if I run the program with `popup` enabled but without a valid chrome instance installed on the machine, it will result in an error.

If the `open` command results in an error, ignore it and change `qrpopuponly` to `true` (i.e the program will not wait for a socket connection to continue to the next task.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.4](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.3...3.1.4) (2020-12-05)

- build(deps): [security] bump highlight.js from 10.2.0 to 10.4.1 [`#1145`](https://github.com/open-wa/wa-automate-nodejs/pull/1145)
- build(deps-dev): bump @types/puppeteer from 5.4.0 to 5.4.1 [`#1139`](https://github.com/open-wa/wa-automate-nodejs/pull/1139)
- build(deps-dev): bump @types/marked from 1.2.0 to 1.2.1 [`#1140`](https://github.com/open-wa/wa-automate-nodejs/pull/1140)
- Release 3.1.4 [`f0621be`](https://github.com/open-wa/wa-automate-nodejs/commit/f0621be229e9759221ddb4dee4f7b2bf85d2d1a9)
- Implemented: `screenshotOnInitializationBrowserError` #1141 [`e79bb0f`](https://github.com/open-wa/wa-automate-nodejs/commit/e79bb0f78e8fa840adbd44eb12f07eb97cc3f185)
- Wait a few seconds on possible TOS_BLOCK to prevent. Take screenshot on error#1117 [`60d579e`](https://github.com/open-wa/wa-automate-nodejs/commit/60d579e09485fa356746ffd7c12a26c2f0ae59f2)
- docs: added linux chrome isntallation docs [`2f23c4d`](https://github.com/open-wa/wa-automate-nodejs/commit/2f23c4db363b2b051be9ab28ac23250e3b85e846)
- Check if program (chrome) is available on machine before trying to open #1142 [`f5e032f`](https://github.com/open-wa/wa-automate-nodejs/commit/f5e032f6ec88ae5c9338260a67d2ab9bdcbe3aae)
- chore: update .npmignore to reduce library package size [`3b16a1d`](https://github.com/open-wa/wa-automate-nodejs/commit/3b16a1d083104c6ce8f1333d5947b07e0177a8b1)
- updated `setChatEphemeral` docs #1049 [`c3f62f4`](https://github.com/open-wa/wa-automate-nodejs/commit/c3f62f417538eafa5e714806739cd2af995b6e3d)
- make sure cname is fixed before release [`b4cd1e5`](https://github.com/open-wa/wa-automate-nodejs/commit/b4cd1e5934f823a64afea28e5b01aba8aac1460b)
- Create CNAME [`d427b45`](https://github.com/open-wa/wa-automate-nodejs/commit/d427b45d9039f7c2bfe99ec2a5647b61f2c256a3)

**Comment by smashah** [comment]
released https://github.com/open-wa/wa-automate-nodejs/releases/tag/3.1.4
